app/lib/forms/choose_your_provider.rb

```diff
@@ -58,6 +58,7 @@ def eligible_for_funding?
         inside_catchment: inside_catchment?,
         new_headteacher: new_headteacher?,
         trn:,
+        get_an_identity_id:,
```

Comment:
whats the reason not getting this from the `query_store` maybe im missing something